src/main/java/com/google/sps/servlets/PrivateRoomServlet.java

```diff
@@ -14,29 +14,24 @@
 
 package com.google.sps.servlets;
 
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
 import com.google.gson.Gson;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that takes in the user's inputted Room ID and directs them
-    to the associated private room once ensurred that it's an actual room*/
-@WebServlet("/join-room")
-public final class JoinRoomServlet extends HttpServlet {
-  private String inputtedUserTag = "user-party-link";
-  private String currentRoomId = "";
+/** Servlet that takes in inputted room Id and directs them to that room
+*/
+@WebServlet("/private-room")
+public final class PrivateRoomServlet extends HttpServlet {
+  private String inputtedIdTag = "user-party-link";
```

Comment:
private static String. Also the field name is not indicative of what you are retrieving from the request.getParameter.

Reply:
Renamed inputted-party-id to make variable name more accurate